### ISSUE TITLE
Is it possible to customize the view of the Library?

### ISSUE THREAD

**Issue by xfinitysoft** [vkurko/calendar#281]
Hi

Thank You for making this great Library.

We need to develop a Booking plugin and we want to know that is it possible to create a similar view like this using this library:
[Link of the reference site](https://liquidspace.com/us/fl/jacksonville/united-states-business-software-llc/office-1?StartTime=&ReservationStart=&WorkspaceCapacity=&Amenities=&SpaceTypes=0&ReservationMethod=0&SortOrder=0&ReservationLengthMinutes=60&src=srp#wp-hourly)

Instead on timeline on left, we want to show days on the left side like this:
[View we need to create](https://i.imgur.com/i0ccMVU.png)

Can we do this?
Can we customize the code to achieve the desired layout?

Your quick response will be highly appreciated.

We look forward to hearing from you soon.
Thank You

**Comment by vkurko** [comment]
No, this view is not available in Event Calendar. I'm closing this issue in favor of #282
